### ISSUE TITLE
Drop sjcl dependency now all supported browsers have built-in sha512

### ISSUE THREAD

**Issue by DanielHeath** [radiopaedia/dicom-anonymiser#30]
`const digest = await window.crypto.subtle.digest('SHA-256', data);`

**Comment by flaki** [comment]
Yeah I had a quick look, and unfortunately the main issue here is "async poisoning", that is, because of the `await` we would need to change every function upstream of `hashedUid()` to be async (and there is no sync API in webcrypto) so I think it's not really possible to drop this without a (not-so-subtle) refactor...

**Comment by DanielHeath** [comment]
Yuck, but probably not worth it then.
